MAUI/ProgressBar/migration.md

```diff
@@ -0,0 +1,587 @@
+<td>Divided into {{'TrackCornerRadius'| markdownify }}, {{'ProgressCornerRadius'| markdownify }} and {{'SecondaryProgressCornerRadius'| markdownify }}</td>
+<td>Gets or sets a value to determine the corner radius of the track, progress and secondary progress.</td></tr>
+</table> 
+
+<table>
+<tr>
+<th>Xamarin</th>
+</tr>
+<tr>
+<td>
+{% tabs %} 
+{% highlight xaml %}
+<ContentPage
+    xmlns:progressBar="clr-namespace:Syncfusion.XForms.ProgressBar;assembly=Syncfusion.SfProgressBar.XForms">
+
+...     
+<progressBar:SfLinearProgressBar CornerRadius="5"
+                                 SecondaryProgressColor="Yellow"
+                                 SecondaryProgress="50"
+                                 >
+
+</progressBar:SfLinearProgressBar>
+
+...
+</ContentPage>
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.XForms.ProgressBar;
+
+...
+
+SfLinearProgressBar linearProgressBar = new SfLinearProgressBar();
+linearProgressBar.CornerRadius = 5;
+linearProgressBar.SecondaryProgressColor = Color.Yellow;
+linearProgressBar.SecondaryProgress = 50;
+this.Content = linearProgressBar;
+
+...
+{% endhighlight %}
+{% endtabs %}
+</td>
+</tr>
+<tr>
+<th>.NET MAUI</th>
+</tr>
+<tr>
+<td>
+{% tabs %} 
+
+{% highlight xaml %}
+
+<ContentPage
+    xmlns:progressBar="clr-namespace:Syncfusion.Maui.ProgressBar;assembly=Syncfusion.Maui.ProgressBar">
+
+...
+<progressBar:SfLinearProgressBar ProgressCornerRadius="5,5,5,5"
+                                 TrackCornerRadius="5,5,5,5"
+                                 SecondaryProgressCornerRadius="5,5,5,5"
+                                 SecondaryProgressFill="Red"
+                                 SecondaryProgress="50"
+                                 >
+    
+</progressBar:SfLinearProgressBar>
+...
+
+</ContentPage>
+ 
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.Maui.ProgressBar;
+...
+
+SfLinearProgressBar linearProgressBar = new SfLinearProgressBar();
+linearProgressBar.ProgressCornerRadius = new CornerRadius(5, 5, 5, 5);
+linearProgressBar.TrackCornerRadius = new CornerRadius(5, 5, 5, 5);
+linearProgressBar.SecondaryProgressCornerRadius = new CornerRadius(5, 5, 5, 5);
+linearProgressBar.SecondaryProgressFill = Colors.Red;
+linearProgressBar.SecondaryProgress = 50;
+this.Content = linearProgressBar;
+... 
+
+{% endhighlight %}
+
+{% endtabs %}
+</td>
+</tr>
+</table>
+
+## SfCircularProgressBar
+
+<table> 
+<tr>
+<th>Xamarin SfProgressBar</th>
+<th>.NET MAUI SfProgressBar</th>
+<th>Description</th></tr>
+<tr>
+<td>{{'[IndicatorInnerRadius](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_IndicatorInnerRadius)'| markdownify }}</td>
+<td>{{'ProgressThickness'| markdownify }}</td>
+<td>Gets or sets a value that specifies the thickness for the progress. You can specify value either in logical pixel or radius factor using the ThicknessUnit property.</td></tr>
```

Comment:
<td>Gets or sets a value that specifies the thickness of the progress. You can specify the value either in logical pixel or radius factor using the ThicknessUnit property.</td></tr>